publish-package.sh

```diff
@@ -5,20 +5,20 @@ set -e
 CURRENT_BRANCH=`git name-rev --name-only HEAD`
 
 if [ $CURRENT_BRANCH != 'master' ] ; then
```

Comment:
Well, this would mean you have to tag locally and push the tag, which is a different release process